snapper/Makefile.am

```diff
@@ -61,9 +61,9 @@ libsnapper_la_SOURCES +=				\
 	Selinux.cc		Selinux.h
 endif
 
-libsnapper_la_CPPFLAGS = $(XML2_CFLAGS) $(ZLIB_CFLAGS)
+libsnapper_la_CPPFLAGS = $(XML2_CFLAGS) $(ZLIB_CFLAGS) $(LIBBTRFSUTIL_CFLAGS)
 libsnapper_la_LDFLAGS = -version-info @LIBVERSION_INFO@
-libsnapper_la_LIBADD = -lboost_thread -lboost_system $(XML2_LIBS) -lacl $(ZLIB_LIBS)
+libsnapper_la_LIBADD = -lboost_thread -lboost_system $(XML2_LIBS) -lacl $(ZLIB_LIBS) $(LIBBTRFSUTIL_LIBS)
```

Comment:
Those additions should only be done if ENABLE_BTRFS is set.

Reply:
Ok, moved under "if ENABLE_BTRFS".